python/templates/Interface.h.jinja2

```diff
@@ -92,10 +92,9 @@ private:
  std::unique_ptr<Concept> m_self{nullptr};
 
 public:
-  template<typename ValueT>
+  template<typename ValueT, typename = std::enable_if_t<isInitializableFrom<ValueT>>> // "{{ class.bare_type }} can only be initialized with one of the following types (and their Mutable counter parts): {{ Types | join(", ") }}");
```

Comment:
Since the comment is quite long, can it be moved to another line?

Reply:
If we move it then it won't be displayed in the compilation error. I put some example in the other suggestion

Reply:
I just realized that anyone who has `clang-format` will probably not see the comment in this form in any case, because that will just split it into several lines, and the error will only contain the first one.

Reply:
Oh, right. I think I'd rather give up and put in another line then